new-client/public/appConfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "appName": "Hajk",
   "version": "3.5.0",
-  "mapserviceBase": "",
+  "mapserviceBase": "http://testservergbg.northeurope.cloudapp.azure.com:82",
```

Comment:
Nah…

Reply:
Sorry, i double checked to not check in my appConfig but i failed

Reply:
Added new commit where i removed the url  https://github.com/hajkmap/Hajk/pull/721/commits/d503d8288b1cd9164b259cd7fdb98ce631fd23e8